.github/workflows/hlint.yaml

```diff
@@ -0,0 +1,22 @@
+name: hlint
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  hlint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: 'Set up HLint'
+      uses: haskell/actions/hlint-setup@v2
+      with:
+        version: '3.4.1'
+
+    - name: 'Run HLint'
+      uses: haskell/actions/hlint-run@v2
+      with:
+        path: XMonad/
```

Comment:
In bd5b969d9ba24236c0d5ef521c0397390dbc4b37 you hlinted the tests as well, and haskell-ci used to hlint everything that ends up in the tarball. Is this change intentional as well?

Reply:
Ah, no, this was me being stupid :)

Reply:
https://github.com/xmonad/xmonad-contrib/commit/bd5b969d9ba24236c0d5ef521c0397390dbc4b37#diff-b376ebbec70caee428a485ef2096195a5f6afe61313e005b15640d8bec912d6d

(sometimes I wonder whether this level of diligence is as annoying as it feels :-))

Reply:
Nono, it's definitely appreciated :)